### PR TITLE
Move some stuff in the right packages

### PR DESCRIPTION
**Pull request by vdemeester**
- This will help split stuff in smaller, better tested packages
- This moves some stuff like the traefik command to package `cmd`

Note that this is a start (and stays small) but the `server` package should be split up then, etc.

/cc @containous/traefik 

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: build.Dockerfile [hunk 1]
```diff
@@ -21,10 +21,10 @@ WORKDIR /go/src/github.com/containous/traefik
 
 COPY glide.yaml glide.yaml
 COPY glide.lock glide.lock
-RUN glide install -v
+RUN glide install --strip-vendor
```
Comment (emilevauge) [inline build.Dockerfile]: @vdemeester why `strip-vendor` ?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -59,11 +59,11 @@ $ glide install --strip-vendor
 # generate (Only required to integrate other components such as web dashboard)
 $ go generate
 # Standard go build
-$ go build
-# Using gox to build multiple platform
-$ gox "linux darwin" "386 amd64 arm" \
-    -output="dist/traefik_{{.OS}}-{{.Arch}}"
+$ go build -o dist/traefik ./cmd/
```
Comment (dtomcej) [inline .github/CONTRIBUTING.md]: Do you really want to specify the output path? the binary is in the gitignore for the root directory. Instead, could you add a shim build in root that just "forwards" to `./cmd`? Sorry for the terrible terminology, but go build is such a simple dev command, that we really should fight to keep ;)

Comment (vdemeester) [inline .github/CONTRIBUTING.md]: Make sense 😛

File: build.Dockerfile (hunk 1 above)
Comment (errm) [inline build.Dockerfile]: -v is the shortcut for --strip-vendor, it makes sense to use the full flag for documentation...

**Comment by emilevauge** [comment]
@vdemeester What do you want to do with this PR? Closing?

**Comment by vdemeester** [comment]
Should close this.. it's too old, I'll redo if I get some time 😝

**Pull request closed without merging**